html/includes/graphs/device/cambium_generic_gpsStats.inc.php

```diff
@@ -19,7 +19,11 @@
     $rrd_options .= ' GPRINT:visible:LAST:%0.2lf%s ';
     $rrd_options .= ' GPRINT:visible:MIN:%0.2lf%s ';
     $rrd_options .= ' GPRINT:visible:MAX:%0.2lf%s\\\l ';
+<<<<<<< HEAD
+    $rrd_options .= " AREA:tracked#00B200:'Tracked       ' ";
+=======
     $rrd_options .= " LINE2:tracked#00ff00:'Tracked       ' ";
+>>>>>>> d0734d78b0d8759e80cd211cdef659e330034682
```

Comment:
Ah shoot, I must have missed removing the file. Thanks. It's not doing anything, thought i removed it.